.github/workflows/c-cpp.yml

```diff
@@ -0,0 +1,32 @@
+name: C/C++ CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: update
+      run: sudo apt update
+    - name: install packages
+      run: sudo apt-get install --no-install-recommends -y debhelper-compat devscripts libapt-pkg-dev gettext libgtk-3-dev libvte-2.91-dev intltool xmlto sharutils libept-dev
```

Comment:
Does "sudo apt build-dep -y ./"  also work here? This would have the nice benefit that it also tests the build-dependencies in debian/control.

Reply:
Yes it does!  I didn't know you could do that :)